app/uk/gov/hmrc/plasticpackagingtax/registration/views/partnerships/check_other_partner_answers_page.scala.html

```diff
@@ -0,0 +1,78 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.Title
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.BackButton
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components._
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.main_template
+@import uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.{routes => partnershipRoutes}
+@import uk.gov.hmrc.plasticpackagingtax.registration.models.registration.UserEnrolmentDetails
+@import uk.gov.hmrc.plasticpackagingtax.registration.models.genericregistration.Partner
+@import java.time.format.DateTimeFormatter
+
+@this(
+        formHelper: FormWithCSRF,
+        govukLayout: main_template,
+        govukButton: GovukButton,
+        govukSummaryList: GovukSummaryList,
+        pageHeading: pageHeading,
+        saveAndContinue: saveAndContinue
+)
+
+@(answers: Partner, backLink: Call)(implicit request: Request[_], messages: Messages)
```

Comment:
This can be replaced with the "official" version now: `partner_check_answers_page`